src/Cake.AzureDevOps/Repos/PullRequest/CommentThread/AzureDevOpsPullRequestCommentThread.cs

```diff
@@ -1,11 +1,11 @@
 ﻿namespace Cake.AzureDevOps.Repos.PullRequest.CommentThread
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
     using Cake.Core.IO;
     using Microsoft.TeamFoundation.SourceControl.WebApi;
     using Microsoft.VisualStudio.Services.WebApi;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
```

Comment:
Revert reordering of usings. We have configured linting rule for `System` namespaces to appear before other usings.

Reply:
Done.